### PR TITLE
Add Yugioh! original to the list, Add my clone project Heroes of Civilizations

### PR DESCRIPTION
**Pull request by francot514**
Added Yugioh! original game to list by creating y.yaml and added my own project that is clone of the game and uses free content.

### CODE REVIEW

File: games/h.yaml [hunk 1]
```diff
@@ -172,6 +172,19 @@
   updated: 2014-05-04
   url: http://hematite.piston.rs/
 
+- name: Heroes of Civilizations
+  images:
+  - https://github.com/francot514/CivHeroes/blob/master/preview.png
```
Comment (cxong) [inline games/h.yaml:177]:
```suggestion
  - https://raw.githubusercontent.com/francot514/CivHeroes/master/preview.png
```

File: games/h.yaml [hunk 2]
```diff
@@ -172,6 +172,19 @@
...
+  license:
+  - GPL2
+  development: active
+  originals:
+  - Yu-Gi-Oh!
+  repo: https://github.com/francot514/CivHeroes
+  type: clone
+  updated: 2020-04-20
+  url: https://github.com/francot514/CivHeroes
```
Comment (cxong) [inline games/h.yaml:186]:
```suggestion
```
same as repo

File: originals/y.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+- name: Yu-Gi-Oh!
+  external:
+    wikipedia: Yu-Gi-Oh!
```
Comment (cxong) [inline originals/y.yaml:3]:
```suggestion
    wikipedia: Yu-Gi-Oh! Trading Card Game
```

**Comment by francot514** [comment]
Updated according to your request

**Pull request merged**